README.rst

```diff
@@ -9,34 +9,38 @@ Here's where to get the code::
     git clone https://github.com/rhinstaller/anaconda-webui.git
     cd anaconda-webui
 
-Running eslint
---------------
+Development instructions
+------------------------
 
-Anaconda Web UI uses `ESLint <https://eslint.org/>`_ to automatically check
-JavaScript code style in `.js` and `.jsx` files.
+See `<HACKING.rst>`_ and `<test/README.rst>`_ for details about how to efficiently change the code,
+run, and test it.
 
-The linter is executed within every build as a plugin of esbuild.
+Automated release
+-----------------
 
-For developer convenience, the ESLint can be started explicitly by::
+The intention is that the only manual step for releasing a project is to create
+a signed tag for the version number, which includes a summary of the noteworthy
+changes::
 
-    npm run eslint
+    123
```

Comment:
Is this a typo?

Reply:
Yes this should not be there, sorry

Reply:
Nope. This is the tag message expected structure. THe eslint part moved in to HACKING.rst.

Reply:
Interesting.